stil22.css

```diff
@@ -0,0 +1,72 @@
+
+
+.split {background-image: url("./žuta.jpg"); 
+margin:2px 3px 25px 3px;
+border-radius:10px; 
+ }
+
+.drugi{background-image: url("./roza.jpg");
+margin:0px 3px 25px 3px;
+border-radius:10px;
+}
+
+.treci {background-image: url("./telefon.jpg");
+    margin:0px 3px 25px 3px;
+    border-radius:10px;
+}
+
+.botun {display:inline-block;
+border-radius:20px;
+border:none;
+background-color:lightgrey;  
+color: white;         
+padding:10px; 
+margin:5px;
+font-family: 'Roboto', serif;
+font-size:14px;
+font-weight:600;
+opacity:0.8;
+}
+
+
+
+.botun2 {display:inline-block;
+    border-radius:20px;
+    border:none;
+    background-color:lightgrey;  
+    color: white;         
+    padding:10px; 
+    margin:5px 5px 25px 5px; 
+   font-family:'Roboto', serif;
+font-size:14px;
+font-weight:600;
+opacity:0.8;
```

Comment:
Instead of `background-color: --` and `opacity: --` you could use `background-color: rgba(0, 0, 0, 0.2)` which makes only bakcground transparent, and not the whole element. Google `rgba` color format.